verify.go

```diff
@@ -116,25 +116,60 @@ func (v *verifier) stop() {
 	v.stopCh <- struct{}{}
 }
 
-func (v *verifier) verifyImages(ctx context.Context, images []string) ([]byte, error) {
-	digests := map[string]string{}
-	for _, image := range images {
-		digest, err := v.verifyImage(ctx, image)
+func (v *verifier) verifyImages(ctx context.Context, images *RequestData) ([]byte, error) {
+	response := ResponseData{
+		Allow:   false,
```

Comment:
can we call this `Verified` instead of `Allow`, as Allow/Deny seems more the the action that  Kyverno will take based on the verification results.